신규-아이디-추천.js

```diff
@@ -0,0 +1,23 @@
+function solution(new_id) {
+    let getTarget = new_id.toLowerCase();
+
+    getTarget = getTarget.match(/[a-z0-9\.\-\_]/g).join("");
+
+    getTarget = getTarget.replace(/\.{2,}/g, ".");
+
+    getTarget = getTarget.replace(/^\./g, "");
```

Comment:
```suggestion
    getTarget = getTarget.replace(/^\.|\.$/g, "");
```
밑에서 끝에 .을 제거해주는 표현식이 있어서 안넣었는지는 모르겠는데 이렇게 하면 앞 뒤를 함께 자를 수 있습니다.